docs/releasing.md

```diff
@@ -79,8 +79,68 @@ The pipeline will:
 
 ## Installer
 
-The Azure Kinect SDK installer provides both open and closed source binaries. The open source binaries are built from 
+The Azure Kinect SDK installer provides both open and closed source binaries. The open source binaries are built from
 this [GitHub repo](https://github.com/Microsoft/Azure-Kinect-Sensor-SDK).
 
-[Release tags](https://github.com/Microsoft/Azure-Kinect-Sensor-SDK/releases) are used by Microsoft to label commits 
+[Release tags](https://github.com/Microsoft/Azure-Kinect-Sensor-SDK/releases) are used by Microsoft to label commits
 used to create a released build. Checkout the commit that matches the release label to build the desired release version.
+
+## Artifacts and Packaging
+The following table contains all artifacts that are released in installers and packages. The file paths listed in the table are only representative, within any given package they will be different to match the standards and conventions for those packages. Each unique released artifact is listed in this table only once, although they will potentially appear duplicate times in packages or installers.
+
+File                                                           | MSI                | NuGet              | libk4a             | libk4a-dev         | libk4a-tools
```

Comment:
What about changing the debian packages to be what they are (like "MSI" and "Nuget") instead of the package name?

```suggestion
File                                                           | MSI                | NuGet              | Runtime Debian Package             | Development Debian Package         | Tooling Debian Package
```